src/main/resources/io/jenkins/plugins/gitlabbranchsource/ProjectNamingStrategyTrait/help-strategyId.html

```diff
@@ -15,5 +15,19 @@
       Other project
       My Subgroup / Third project
     </dd>
+    <dt>Contextual Project Path</dt>
+    <dd>
+      This option will use the project path with nested groups and project, but not the owner segment. Example:
+      my-project
+      other-project
+      my-subgroup/third-project
+    </dd>
+    <dt>Simple Project Path</dt>
+    <dd>
+      This option will use the project path with no groups. Example:
+      my-project
+      other-project
+      third-project
+    </dd>
```

Comment:
Would appreciate an better example.

I am unsure how this would show up in the UI.

The screenshot you took does not really demonstrate the difference.

That is to say the original examples might not be good either 😖

Reply:
The difference is the original naming strategy includes all the nested groups in the name of the object.  I'm not sure how else to describe the problem or provide a better example.  Note how "third-project" does not include "my-subgroup/" in it.

Reply:
As for the example screen shot.  If not for the new naming strategy, each of those would have had "mock/" in from of the name of each repository because that is the name of the "owner" (group).  Since the purpose of this plugin is the scan repositories in a group, you must specify an "owner" group to scan.  The way the default naming strategy works, that same owner group will always be included in the name of each found project in Jenkins.  So, if the owner is "mock" then every single project found inside it is also prefixed with "mock/".

Reply:
Is this any better?
![image](https://user-images.githubusercontent.com/2965467/163823505-d97ab887-0060-4cfb-88b1-13699e304eeb.png)

Reply:
Better I guess I cannot read.

Reply:
No worries.  The differences (especially with "Contextual Project Path") can be subtle.  The new formatting definitely helps.